daml/Marketplace/Utils.daml

```diff
@@ -7,6 +7,20 @@ import DA.Next.Set
 import DA.Optional
 import DA.Text
 
+template DismissibleNotification
+  with
+    sender : Party
+    senderRole : Optional MarketRole
+    receiver : Party
+    text : Text
+  where
+    signatory sender, receiver
```

Comment:
I would argue that the signatory of a notification should be the sender only.

Reply:
I think you're right, I'll change the `receiver` to be an `observer`.

Reply:
I didn't think the `Operator` needed to have anything to do with `Notifications`, but I'm noticing that the majority of templates do have the `operator` have a field, and many with the `operator` as a `signatory`. Would there be a reason to add the `operator` here?